src/EA.Weee.Web/Areas/Admin/Controllers/AatfReportsController.cs

```diff
@@ -425,7 +425,7 @@ private async Task PopulateFilters(AatfAeDetailsViewModel model)
         {
             using (var client = apiClient())
             {
-                model.ComplianceYears = new SelectList(FetchAllAatfComplianceYears());
+                model.ComplianceYears = new SelectList(new List<int> { 2019, 2020, 2021 });
```

Comment:
Think we would need a new request handler for this... should query all the unique Compliance years in the AATF table